src/api/apiData.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+import shallowEqualSelector from '../helper/shallowEqualSelector';
+import tableHeights from '../selectors/tableHeights';
+
+const getApiDataSelector = () =>
+  shallowEqualSelector(
+    [
+      (state) => state.tableSize.height,
+      (state) => state.elementHeights.groupHeaderHeight,
+      (state) => state.scrollX,
+      (state) => state.maxScrollX,
+      (state) => tableHeights(state).availableScrollWidth,
```

Comment:
availableScrollWidth is now calculated through the `tableHeights` selector instead of the `columnWidths` selector (which was present in v2).
(For context: `columnWidths` selector relied on getting all column info available, and hence was removed in the column virtualization branch)